src/open_inwoner/templates/mail/_base.html

```diff
@@ -137,7 +137,7 @@
 
     .button--primary {
         background-color: {{ theming.primary_color }};
-        border: 1px solid {{ theming.primary_color }};
```

Comment:
Was this space intentionally removed?

Reply:
Ew, it was sadness from cherrypicking... Will revert.